tools/travis/setup.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash
 
+sudo gpasswd -a travis docker
+sudo -E bash -c 'echo '\''DOCKER_OPTS="-H tcp://0.0.0.0:4243 -H unix:///var/run/docker.sock --api-enable-cors --storage-driver=aufs"'\'' > /etc/default/docker'
+
 # Docker
 sudo apt-get -y update -qq
-sudo apt-get -o Dpkg::Options::="--force-confnew" --force-yes -y install docker-engine=1.9.1-0~trusty
+sudo apt-get -o Dpkg::Options::="--force-confold" --force-yes -y install docker-engine=1.9.1-0~trusty
```

Comment:
Is the only  fix is `--force-confold`?  Or also we need to use `environment/local` also on top of it?